runtime/src/main/java/org/corfudb/runtime/collections/CorfuTable.java

```diff
@@ -390,6 +393,13 @@ protected void undoPut(CorfuTable<K, V> table, V undoRecord, K key, V value) {
         undoRemove(table, undoRecord, key);
     }
 
+    @DontInstrument
+    protected List<Object> identifyPutGarbage(Object locator, K key, V value) {
+        List<Object> garbage = new ArrayList<>();
+        garbage.addAll(locatorStore.addUnsafe(key, (SMRRecordLocator) locator));
```

Comment:
You can use `List<Object> garbage = new ArrayList<>(locatorStore.addUnsafe(key, (SMRRecordLocator) locator));` Also you can make it `List< SMRRecordLocator >` as suggested in the previous comment

Reply:
The code is modified to make it more concise. 
I can't change Object to SMRRecordLocator.